src/test/java/com/google/devtools/build/lib/rules/python/PyBaseConfiguredTargetTestBase.java

```diff
@@ -142,6 +142,36 @@ public void srcsPackageNameCannotHaveHyphen() throws Exception {
         "    srcs = ['foo.py', '//pkg-hyphenated:bar.py'])");
   }
 
+  @Test
+  public void packageNameCanHaveHyphenIfImports() throws Exception {
+    scratchConfiguredTarget("pkg-hyphenated", "foo",
+            ruleName + "(",
+            "    name = 'foo',",
+            "    srcs = ['foo.py'],",
+            "    imports = ['.'],",
+            ")");
+    assertNoEvents();
+  }
+
+  @Test
+  public void depsPackageNameCanHaveHyphenIfImports() throws Exception {
```

Comment:
I want to test both, test resources permitting. You're correct that `deps` doesn't have its own separate restriction today, but the test case is intended to guard against regressions in case someone tries to add such a check in the future.

Reply:
It just seems strange that this PR introduces a test case for the existing behavior of deps, but not for the new behavior of srcs. (There's already a test for the old behavior of srcs when no `imports` attr is present.)